src/schema/nmdc.yaml

```diff
@@ -350,11 +378,16 @@ classes:
   omics processing:
     aliases:
       - omics assay
+      - sequencing project
+      - experiment
```

Comment:
`experiment` seems a little too broad to be an alias.

Reply:
see @dehays comment. We should probably distinguish broad/narrow/exact synonyms as we do in OBO, but this is mostly here to index searching on the schema for now

Reply:
Hmmm ... so, should we allow broad synonyms as aliases? Perhaps it might be better to limit aliases to exact synonyms?